release_creation/snapshot/requirements/v1.5.latest.requirements.txt

```diff
@@ -0,0 +1,12 @@
+dbt-core~=1.4.0 --no-binary dbt-postgres
+dbt-snowflake~=1.5.0
+dbt-bigquery~=1.5.0
+dbt-redshift~=1.5.1
+dbt-postgres~=1.5.0
+dbt-spark[PyHive,ODBC]~=1.5.0
+dbt-databricks~=1.5.0
+dbt-trino~=1.5.0
+dbt-rpc~=0.1.1
```

Comment:
`dbt-rpc` is on 0.4.1 now.  @ChenyuLInx what's the lowest version of dbt-rpc that we should use with 1.5?

Reply:
I added 0.4.1 for now. I'll wait for @ChenyuLInx's response and change if needed.

Reply:
0.4.0